tests/unit/test_loaders.py

```diff
@@ -128,6 +129,23 @@ def load_file(self, name):
         loaded = loader.load_data('baz')
         self.assertEqual(loaded, ['loaded data'])
 
+    @mock.patch('os.path.isdir', mock.Mock(return_value=True))
+    def test_load_data_optionally_returns_path(self):
+        search_paths = ['foo', 'bar', 'baz']
+
+        class FakeLoader:
+            def load_file(self, name):
+                expected_ending = os.path.join('bar', 'abc')
+                if name.endswith(expected_ending):
+                    return ['loaded data']
+
+        loader = Loader(
+            extra_search_paths=search_paths, file_loader=FakeLoader()
+        )
+        loaded, path = loader.load_data('abc', return_path=True)
+        self.assertEqual(loaded, ['loaded data'])
+        self.assertEqual(path, 'bar/abc')
```

Comment:
you'll need to update this to `os.path.join('bar', 'abc')` or `f'bar{os.sep}abc'` to pass windows tests